lib/tasks/hourglass.rake

```diff
@@ -45,4 +45,11 @@ namespace :hourglass do
     date = args[:date] ? Date.parse(args[:date]) : Date.current.last_month
     puts GenerateMonthlyReport.call!(date: date).output
   end
+
+  desc "Send report on current weekly team hours usage"
+  task :team_hours_update => :environment do
+    if Date.current.wednesday? || Date.current.friday?
+      SendTeamHoursUpdate.call
```

Comment:
Using `call!` from rake tasks is good practice because `call!` will raise an error if the interactor fails.